packages/mantine/src/components/table/Table.tsx

```diff
@@ -55,6 +56,7 @@ export const Table: TableType = <T,>({
     const convertedChildren = Children.toArray(children) as ReactElement[];
     const header = convertedChildren.find((child) => child.type === TableHeader);
     const footer = convertedChildren.find((child) => child.type === TableFooter);
+    const refresher = convertedChildren.find((child) => child.type === TableRefresher);
```

Comment:
I would just change the naming a bit to something more generic, like `Table.WithinContext`, `Table.ContextConsumer`, `Table.Children`, or any other idea that explains that its for components that need the table context.

Reply:
I went with Table.Consumer